src/api/EpiData.ts

```diff
@@ -182,18 +186,24 @@ export function importCOVIDcast({
 }: {
   data_source: string;
   signal: string;
-  time_type?: 'day';
+  time_type?: string;
   geo_type: string;
   geo_value: string;
 }): Promise<DataGroup | null> {
-  const title = `[API] Delphi CODIDcast: ${geo_value} ${signal} (${data_source})`;
+  const title = `[API] Delphi COVIDcast: ${geo_value} ${signal} (${data_source}) ${time_type}`;
```

Comment:
This line is cramped and busy enough already -- can we improve it?  I dont think the `time_type` is very useful here, and is actually kinda noisy.  How about:
```suggestion
  const title = `[API] COVIDcast: ${data_source}:${signal} (${geo_type}:${geo_value})`;
```